features/support/api_helpers.rb

```diff
@@ -52,15 +56,15 @@ def ensure_user_exists(user_details)
   creation_response = call_api(:post, "/users", payload: {
     users: user_details,
     updated_by: "functional tests",
-  })
+  }, safe_for_smoke_tests: true)
```

Comment:
Is this one safe for the smoke tests? It could actually create a user with this first call right?